packages/page-staking/src/Payouts/index.tsx

```diff
@@ -184,15 +184,15 @@ function Payouts ({ className = '', isInElection, ownValidators }: Props): React
   );
 
   const headerStashes = useMemo(() => [
-    [myStashesIndex ? t('payout/stash') : t('overall/validator'), 'start', 2],
+    [t('account name'), 'start', 2],
```

Comment:
```suggestion
    [t('stash'), 'start', 2],
```
(though perhaps it's obvious enough as is)

Reply:
✅